__tests__/app.test.js

```diff
@@ -52,4 +52,38 @@ describe("GET", () => {
             })
         })
     })
+    describe("/api/article/:article_id", () => {
+        test("200 - returns single article by article id", () => {
+            return request(app)
+            .get("/api/article/1")
+            .expect(200)
+            .then(({body: {article}}) => {
+                expect(Object.keys(article).length).toBe(8)
```

Comment:
Checking the number of properties is not necessary, unless you want to prohibit any further expansion of this object with new properties.